### PR TITLE
Ports one outfit from The Girly Station

### PR DESCRIPTION
**Pull request by Krysonism**
## About The Pull Request
This PR adds the pink summer skirt to the clothesmate.

This outfit was sprited by me, intended for the girly station.

After showing my work to the good people of discord they convinced me there was enough demand for me to attempt to add the outfit to /TG/ proper.

Thanks to Myst Leissa for inspiring me to create this outfit.

## Why It's Good For The Game

Spacewomen need some appropriate clothing in order to not overheat during the space summer.
It costs 75 credits so it also integrated with the goofconomy.

summer
summer_side

## Changelog
:cl:
add: a new skirt has been added to the clothesmate!
imageadd: added sprites for the new outfit.
/:cl:

### CODE REVIEW

**Comment by Fikou** [comment]
b ruh

**Review by optimumtact — APPROVED** [review]

**Comment by Arathianen** [comment]
Wew

**Comment by Akrilla** [comment]
we like girlystation

**Comment by pireamaineach** [comment]
What a utilitarian outfit. The freedom of movement!

**Comment by MMMiracles** [comment]
oh no

**Comment by TerraGS** [comment]
port girly station

**Comment by 81Denton** [comment]
Finally a proper uniform for sec mains

**Comment by Jordie0608** [comment]
angrybird

**Comment by GuyonBroadway** [comment]
It's just like a roman military uniform!



Tex if you are reading this I apologise.

**Comment by Krysonism** [comment]
I see we have some doubters and dissenters in this PR. That is okay, even Jesus had his detractors.

I haven't really heard any substantive negative feedback but I think this is because there is no male version of the outfit yet. 
I will propose such a version once this gets merged and I find time to adjust the chest area to better conform to male anatomy.

**Comment by PKPenguin321** [comment]
Is this the first outfit we've had that enlarges the breasts? I think it may be the first I've seen

**Comment by GuyonBroadway** [comment]
I think the shades of pink are ugly, the pillow shading is ugly and the copy pasting from the jumpskirts is lazy and ugly.

**Comment by GuyonBroadway** [comment]
Oh geeze taking a closer look at the side sprite those oval "boobs" look like oversized jam doughnuts glued to the torso at an angle and the underneath sprite sticks out in a way that doesn't make sense.

**Comment by MMMiracles** [comment]
hey can you also make the hit-or-miss meme skirt while you're at it

**Comment by NothingIsOkay1** [comment]
''the girly station'' is probably the tranniest name ive ever heard, i can only imagine what goes in there

**Comment by pireamaineach** [comment]
Please stop injecting inherently sexual outfits into /tg/ station.
Jumpskirts are one thing but this is deserving of God's wrath.
Especially since you compared yourself to Jesus because people didn't like it.

**Comment by Krysonism** [comment]
This is not really pillow shaded. While shading could be improved in some areas I still think the sprite quality is quite good.

I also don't think it is fair to characterize it as a lazy copy and paste job. 

If you pull up uniforms.dmi you will see that it is not substantially similar to any other sprite.

**Comment by GuyonBroadway** [comment]
The actual skirt part is clearly copied from the skirts of other jumpskirts.

**Comment by GuyonBroadway** [comment]
Also its a good 2px longer side on on the skirt than it is facing front.

**Comment by kriskog** [comment]
Not lockable, that's a downthumb from me.

**Comment by blurbo** [comment]
any post with the word girly or outfit will probably get approved by v*ranges
optimumtact approved these changes X hours ago

**Comment by pireamaineach** [comment]
I just want to point out that this is actually called a, "Bimbo Skirt" by yourself.
https://github.com/MystLeissa/The-Girly-Station/pull/1
Weird that you decided to omit that fact in this P.R. hmmmmmmm

**Comment by Krysonism** [comment]
> 
> 
> I just want to point out that this is actually called a, "Bimbo Skirt" by yourself.
> [MystLeissa#1](https://github.com/MystLeissa/The-Girly-Station/pull/1)
> Weird that you decided to omit that fact in this P.R. hmmmmmmm

I don't hide that fact at all, i asked in the discord for a more /tg/ appropriate name. 

I am a big believer in choosing good names for new features which i am sure Cobby can attest to. I chose to respect the codebase culture of the girly station by selecting a name which the head coder would find agreeable.

I find it a bit crass that you would impugn my motives, but on the other hand it is not really a criticism of the feature itself and i have faith that the good people of github will see through this tactic.

I think, that unless you are a very sheltered person you would not find it that unusual if you saw a woman wearing such an outfit on hot summer day in Berlin or London(I don't know about Dublin, never been).

**Comment by pireamaineach** [comment]
I don't know how hot you think it is in London or Berlin but if you think any respectable person would be caught dead in this, you're a nutter. If it was hot enough to justify this outfit you'd get 1st degree burns by standing outside for a couple minutes.

**Comment by Krysonism** [comment]
It surprises me that someone could take offence to seeing an "immodest dress" in a video game for adults.

This is hardly the first cleavage in sci-fi media.

Imagine someone going on an image board and complaining about seeing a woman in somewhat revealing clothes in Deus Ex or the fifth element! They would be mocked for their opinion and labelled puritans!

**Comment by GuyonBroadway** [comment]
Ok cut the meming and tell us how much oranges paid you for this joke pr...

**Comment by Jordie0608** [comment]
Nuke ops are red
Comdoms are blue
This PR is trash
And the sprite is too

**Comment by nicbn** [comment]
That thread is absolute gold

**Comment by Krysonism** [comment]
Lazarus, come forth!

**Comment by granpawalton** [comment]
> 
> 
> Is this the first outfit we've had that enlarges the breasts? I think it may be the first I've seen

I believe the sec skirt also does boob sprites

**Review by 81Denton — COMMENTED** [review]
lol

**Comment by Repukan** [comment]
Oppai

**Comment by zombiegums** [comment]
lmao

i want to see this on a male sprite

**Comment by wesoda25** [comment]
What the fuck

**Comment by GuyonBroadway** [comment]
Its a meeeeem.

**Comment by ExcessiveUseOfCobblestone** [comment]
I think this affects my roleplaying capabilities when my character is, in fact, an A cup.

**Comment by Difrit** [comment]
To the void with you, degenerate

**Comment by PKPenguin321** [comment]
>Is this the first outfit we've had that enlarges the breasts? I think it may be the first I've seen

>I believe the sec skirt also does boob sprites

So are we gonna make the male version give you a big veiny cock or what

**Comment by 81Denton** [comment]
Add a male version that makes me look like Kenshiro or bust

File: code/modules/clothing/under/skirt_dress.dm [hunk 1]
```diff
@@ -99,3 +99,13 @@
...
+
+/obj/item/clothing/under/dress/skirt/summer
+	desc = "A revealing outfit perfect for a warm summer day."
+	custom_price = 75
+
+/obj/item/clothing/under/dress/skirt/summer/pink
+	name = "pink summer skirt"
+	icon_state = "pink_summer_skirt"
+	item_state = "pink_summer_skirt"
+	item_color = "pink_summer_skirt"
```
Comment (Mickyan) [inline code/modules/clothing/under/skirt_dress.dm]: actioninja is going to love this

**Review by Mickyan — COMMENTED** [review]
maybe I'm the fool but I'll bite

File: code/modules/clothing/under/skirt_dress.dm [hunk 2]
```diff
@@ -99,3 +99,13 @@
 	icon_state = "plaid_green"
 	item_state = "plaid_green"
 	item_color = "plaid_green"
+
+/obj/item/clothing/under/dress/skirt/summer
+	desc = "A revealing outfit perfect for a warm summer day."
+	custom_price = 75
```
Comment (Mickyan) [inline code/modules/clothing/under/skirt_dress.dm]: what is this bogus type with no icon state and why use a custom price if you're going to use the standard vendor price anyway

**Comment by Repukan** [comment]
W-whats

What's the girly station

**Comment by optimumtact** [comment]
this guy made lockable schoolgirl uniforms and pred them to tg and when they got denied he made his own fork

**Comment by Repukan** [comment]
Bruh

**Comment by KetasDotepasto** [comment]
Lockable schoolgirl uniform sounds like it would be the perfect grief item. Doesn't kill you, but makes you wish it did.

**Comment by Krysonism** [comment]
Orange man why! I made this PR just for you baby.

**Comment by optimumtact** [comment]
jebaited

**Comment by ExcessiveUseOfCobblestone** [comment]
PR this as a byond member perk

**Comment by bobbahbrown** [comment]
good pr

**Comment by Naloacisamazing** [comment]
based

**Comment by PKPenguin321** [comment]
https://github.com/tgstation/tgstation/pull/46113 reopen this one too please

**Comment by optimumtact** [comment]
superseded by #49500

**Pull request closed without merging**